pkg/actions/service.go

```diff
@@ -31,8 +32,9 @@ const (
 )
 
 var (
-	ErrNotFound = errors.New("not found")
-	ErrNilValue = errors.New("nil value")
+	ErrNotFound      = errors.New("not found")
```

Comment:
Do we need to control the new error types somewhere?

Reply:
What do you mean in control? the errors will be reflected like other errors that each action may fail with.

Reply:
Talked f2f. Errors in actions are just written to the event